tests/tasks/test_daily_tasks.py

```diff
@@ -61,6 +61,10 @@
 import pytest
 from six.moves import range
 from django.contrib.sites.models import Site
+from waffle.testutils import override_switch
+from mock import patch
+import logging
```

Comment:
From PEP8 (https://www.python.org/dev/peps/pep-0008/#imports)

> Imports should be grouped in the following order:
>
>    Standard library imports.
>    Related third party imports.
>    Local application/library specific imports.
>
> You should put a blank line between each group of imports.

So `import logging` should go above `import pytest` since `logging` is part of the standard library. The existing code (before your PR) does not have a blank line between the following, so while you reorder the imports, please add a blank line there.

```
from datetime import date
import pytest
```

Although I haven't seen it as part of PEP8, I generally order imports alphabetically within each group with the purpose to speed up visually scanning imports within a group. Admittedly, I'm not 100% consistent on this. But when I see it in my own code I've previously written, I try to fix it

 (but I'm not 100% consistent and don't want to be prescriptive here)